src/python_testing/TC_FAN_3_5.py

```
@@ -21,208 +21,1100 @@
+            await self.send_single_cmd(cmd, endpoint=self.endpoint)
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, Status.Success, f"[FC] Unexpected error returned ({e})")
+            pass
 
-        self.print_step("9c", "TH sends Step command to DUT with Direction set to Decrease, Wrap set to True and LowestOff set to True")
-        await self.send_step_command(endpoint=endpoint, direction=Clusters.Objects.FanControl.Enums.StepDirectionEnum.kDecrease, wrap=True, lowestOff=True)
+    async def subscribe_to_attributes(self, handle_current_values: bool) -> None:
+        cluster = Clusters.FanControl
+        attr = cluster.Attributes
+
+        if hasattr(self, 'subscriptions') and self.subscriptions is not None:
```

Pylint (4.0.7)

[error] Access to member 'subscriptions' before its definition line 351

(E0203)